Cli/SendReports.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Authors: Alex Gusev <alex@flancer64.com>
+ * Since: 2020
+ */
+
+namespace Flancer32\Csp\Cli;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Analyze CSP violation reports and compose policy rules.
+ */
+class SendReports
+    extends \Flancer32\Base\App\Cli\Base
+{
+    private const DESC = 'reports new rules to specified mail addresses and registers reported rules persistently in protocol table';
+    private const NAME = 'fl32:csp:report_new_rules';
+
+    /** @var \Flancer32\Csp\Helper\Config */
+    private $hlpCfg;
+    /**
+     * @var \Flancer32\Csp\Model\Service\ReportNewRules
+     */
+    private $reportNewRules;
+
+    public function __construct(
```

Comment:
please turn the constructor arguments into proxies (`\Proxy` suffix`) to improve console startup time and prevent bugs

Reply:
great to have the reason, thanks